cluster/log-dump.sh

```diff
@@ -248,6 +248,9 @@ function dump_nodes() {
 }
 
 setup
-echo "Dumping master and node logs to ${report_dir}"
+echo "Dumping master logs to ${report_dir}"
 dump_masters
-dump_nodes
+if [[ "${DUMP_ONLY_MASTER_LOGS:-}" != "true" ]]; then
```

Comment:
Consider adding an else statement to make it clear we are intentionally not dumping node logs